src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_azcopy_scenarios.py

```diff
@@ -106,9 +115,6 @@ def test_storage_blob_azcopy_remove(self, resource_group, storage_account_info,
         self.cmd('storage remove -c {} -n readme --account-name {}'.format(
             container, storage_account))
 
-        self.cmd('storage remove -c {} -n readme --account-name {}'.format(
-            container, storage_account), expect_failure=True)
```

Comment:
still keep this command and remove "expect_failure"

Reply:
done